src/main/java/net/imglib2/type/operators/PowFloatingPoint.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * #%L
+ * ImgLib2: a general-purpose, multidimensional image processing library.
+ * %%
+ * Copyright (C) 2009 - 2018 Tobias Pietzsch, Stephan Preibisch, Stephan Saalfeld,
```

Comment:
Running `mvn license:update-file-header` should automatically update this to 2020.

Reply:
Thanks, fixed, rebased.